regolith/helpers/u_milestonehelper.py

```diff
@@ -123,12 +124,15 @@ def db_updater(self):
         if rc.index > 1:
             doc = all_milestones[rc.index-2]
             identifier = doc['identifier']
+            if rc.type:
+                doc.update({'type': ALLOWED_TYPES[rc.type]})
+            else:
+                if not doc.get('type'):
+                    raise RuntimeError(f"please rerun specifying --t with a value from {ALLOWED_TYPES}")
```

Comment:
what if it is an update and 'type' is already specified.  Do we want to crash out or just leave the original type?  I think yes.  Please check you code behaves that wya.

Reply:
I checked it, doc is the milestone that is being updated. Thus, if doc has "type", it will just keep going. I modified the code a little though.